src/main/scala/edu/ie3/simona/model/participant/PvModel.scala

```diff
@@ -551,12 +581,12 @@ final case class PvModel private (
 
     if (eDifH.value.doubleValue > 0) {
       // if we have diffuse radiation on horizontal surface we have to check if we have another epsilon due to clouds get the epsilon
-      var epsilon = ((eDifH + eBeamH) / eDifH +
+      var epsilon = ((eDifH + eBeamH / cos(thetaZInRad)) / eDifH +
```

Comment:
![image](https://github.com/ie3-institute/simona/assets/149690068/ffe73d60-24aa-412b-90c3-7b74a1a29522)

Reply:
This is my reference. Note that "I_b,n" is not the beam radiation on a horizontal surface, but rather the beam radiation on a surface normal to the direction of the sun. This can be calculated by dividing the beam radiation on a horizontal surface "I_b" (=eBeamH in code) by cos(ThetaZ). This is also demonstrated in Duffie: example 2.16.2

Reply:
I think we need some more investigation here, probably focused on the weather data specification. I_b,n is certainly the "normal beam incidence radiation", which means the radiation in direction of the beam, i.e. on a surface that is exactly facing the sun, as you explained. I_b is certainly the radiationon a horizontal plane (i.e. on earth's surface).
All of this is illustrated by figure 1.8.1 in Duffie (4th ed). 

The question now becomes which type of radiation the weather data sources provide. For ERA5, [solar radiation always seems to be specified on a horizontal plane](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview). I have not found the specs for ICON and COSMO yet.